### DISCUSSION TITLE
[DEV] Free MySQL Host with read-only access

### DISCUSSION THREAD

**Discussion by FrancescoBorzi** [azerothcore/azerothcore-wotlk#3957] [General]
I'm looking for a MySQL hosting service with the following features:

1. Should be completely **free**. I don't care about performance or other stuff as it will be for demo purposes only
2. Should accept a large databases (I plan to upload the AC world database, ~450mb)
3. Should allow me to create a user with read-only access (only `SELECT` allowed, no `INSERT/UPDATE/DELETE` or other queries) that I can share with the public

An alternative to point 3. it would be super cool if it could have both read and write access but some kind of automatic "reset" mechanism that resets all data to a default value periodically.

**Comment by Viste** [comment]
send you variant via pm in Discord
